pootle/apps/pootle_store/constants.py

```diff
@@ -73,3 +73,7 @@
     UNTRANSLATED: "untranslated",
     FUZZY: "fuzzy",
     TRANSLATED: "translated"}
+
+# Default store priority - used by vfolders atm
+#  - valid range = 0 < n <= 999.99
```

Comment:
It would be great to enforce it as we do for lower limit. But lets leave that for further PR.